docs/source/start.rst

```diff
@@ -12,27 +20,26 @@ We recommend installing pyQuil using package manager pip.
 
 .. code::
 
-    `pip install pyquil`
+    pip install pyquil --pre
 
-will install pyQuil you can install requirements directly by typing
+will install pyQuil. You can install requirements directly by typing
 
 .. code::
 
-    `pip install -r requirements.txt`
+    pip install -r requirements.txt
 
-in your pyquil folder.
-
-For those of you that already have pyQuil, you can upgrade by typing
+in your pyquil folder. For those of you that already have pyQuil, you can upgrade by typing
 
 .. code::
 
-    `pip install --upgrade pyquil`
+    pip install --upgrade pyquil --pre
```

Comment:
although it still may work, I think it makes more sense (and matches the usage instructions) to put the `--pre` before `pyquil` as it is a member of `[options]`
```
peter@peter-macbookpro ➜  pip install -h

Usage:
  pip install [options] <requirement specifier> [package-index-options] ...
  pip install [options] -r <requirements file> [package-index-options] ...
  pip install [options] [-e] <vcs project url> ...
  pip install [options] [-e] <local project path> ...
  pip install [options] <archive url/path> ...
```